patches/extra/inox-patchset/0008-restore-classic-ntp.patch

```diff
@@ -47,4 +47,4 @@
 +    "KeepPrefetchedContentSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
  
  const base::Feature kOptionalImagesEnabledFeature{
-     "NTPRemoteSuggestionsOptionalImages", base::FEATURE_DISABLED_BY_DEFAULT};
+     "NTPRemoteSuggestionsOptionalImages", base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
What does changing this flag default do?

Reply:
Looks like I missed a flag. I just updated the patch to disable that feature.